IFC4x3/Templates/Object Composition/Surface Feature Adherence/DocTemplateDefinition.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocTemplateDefinition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Surface Feature Adherence" UniqueId="db1e5cb3-9a56-42c0-9633-90e92a89eff6" Type="IfcElement">
+	<Rules>
+		<DocModelRuleAttribute Name="HasSurfaceFeatures">
+			<Rules>
+				<DocModelRuleEntity Name="IfcRelAdheresToElement">
+					<Rules>
+						<DocModelRuleAttribute Name="RelatedSurfaceFeatures">
+							<Rules>
+								<DocModelRuleEntity Name="IfcSurfaceFeature">
+									<Rules>
+										<DocModelRuleAttribute Name="Name">
```

Comment:
Doesn't one need `Identification` to be able to specify this as a `RuleID` in the mvdXML?

Reply:
Addressed
@SergejMuhic @jmirtsch I can't seem to find how to add this in the interface??